core/backend/package.json

```diff
@@ -93,10 +94,12 @@
     "sinon": "^17.0.2",
     "source-map-loader": "^4.0.0",
     "typescript": "~5.6.2",
+    "marked": "^14.1.3",
```

Comment:
any reason we are not using v15?

Reply:
No reason. 15 was released a week after I created the prototype for these tests, and we never updated that's all.